packages/pwa-kit-react-sdk/src/ssr/universal/hooks/index.js

```diff
@@ -27,22 +27,20 @@ export const useCorrelationId = () => {
  * @typedef {Object} ServerContext
  * @property {Object} req - Request object
  * @property {Object} res - Response object
- * @property {boolean} isServerSide
  */
 
 /**
  * Get the server context
  * @returns {ServerContext} ServerContext object
  *
  * @example
- * const {res, isServerSide} = useServerContext()
- * if (isServerSide && query.error) { res.status(404) }
+ * const {res} = useServerContext()
+ * if (res && query.error) { res.status(404) }
  */
 export const useServerContext = () => {
     const serverContext = useContext(ServerContext)
 
     return {
-        ...serverContext,
-        isServerSide: Boolean(serverContext.req)
+        ...serverContext
```

Comment:
Minor: now that we're no longer extending the object, we can simply return the context directly like...

```js
return useContext(ServerContext)
```